### ISSUE TITLE
can't set a variable

### ISSUE THREAD

**Issue by Kasst1el** [avocode/avocode#1668]
* **Version**: 2.16.0

* **Web/Desktop**: Desktop

* **Platform**: win32

* **Avocode UID**: 165211
---

```
Error: ProjectApi#getProjectById() -> error (status: 404, error: "Not found")
    at t (file:///C:/Users/%D0%90%D1%80%D1%82%D0%B5%D0%BC/AppData/Local/avocode/app-2.16.0/resources/app.asar/dist/avocode.min.js:1:145283)
    at t (file:///C:/Users/%D0%90%D1%80%D1%82%D0%B5%D0%BC/AppData/Local/avocode/app-2.16.0/resources/app.asar/dist/avocode.min.js:1:145174)
    at new t (file:///C:/Users/%D0%90%D1%80%D1%82%D0%B5%D0%BC/AppData/Local/avocode/app-2.16.0/resources/app.asar/dist/avocode.min.js:1:145817)
    at Object._getErrorByStatusCode (file:///C:/Users/%D0%90%D1%80%D1%82%D0%B5%D0%BC/AppData/Local/avocode/app-2.16.0/resources/app.asar/dist/avocode.min.js:1:39208)
    at Object.normalizeError (file:///C:/Users/%D0%90%D1%80%D1%82%D0%B5%D0%BC/AppData/Local/avocode/app-2.16.0/resources/app.asar/dist/avocode.min.js:1:38738)
    at Object.normalizeResponse (file:///C:/Users/%D0%90%D1%80%D1%82%D0%B5%D0%BC/AppData/Local/avocode/app-2.16.0/resources/app.asar/dist/avocode.min.js:1:38569)
    at t.no
```

**Comment by jakubzitny** [comment]
Thanks for the report @Kasst1el, this is a bug and we'll try to fix it asap. Do you have any more details on how did it happen? I believe it only happened once, right? When you restart the app, login/logout or just re-open the design, the variables should work correctly.

**Comment by jakubzitny** [comment]
Hi, this has been fixed in latest Avocode `2.16.1`, please update and let us know if you have any more problems :wink:.
